### DISCUSSION TITLE
How to do arithmetic of anchors?

### DISCUSSION THREAD

**Discussion by Kreijstal** [cetz-package/cetz#137] [Q&A] [answered]
if you name an anchor "t"
and then do "t.right" how do you add to that or extract the coordinate, for example "t.right"+(1,2)?

**Comment by johannes-wolf** [comment, marked as answer]
Using relative coordinates:
`(rel: (1, 2), to: "l.right")`

**Reply by Kreijstal** [reply]
what if you want to get the middle point of 2 points as in
there are anchor "a" and "b"

and you want to do "a"+("b"-"a")/2

with relative it only works if you want to add a literal with a variable coordinate..

**Reply by johannes-wolf** [reply]
Linear interpolation between points can be done via the form:
`(start, t, end)`, so the midpoint would be `(a, 0.5, b)`.
